Sources/Get/get().swift

```diff
@@ -17,9 +17,9 @@ import Utility
  - Throws: Error.InvalidDependencyGraph
  - Returns: The modules that this manifest requires building
 */
-public func get(manifest: Manifest) throws -> [Package] {
+public func get(manifest: Manifest, manifestParser: (path: String, url: String) throws -> Manifest) throws -> [Package] {
```

Comment:
Agree with Max. I've used `typealias` for function types in production app, didn't like it. I think it's worse also because autocompletion just five you `ManifestParser` instead of `(path: String, url: String) throws -> Manifest`